packages/solid-crs-pods/data/.acl

```diff
@@ -11,3 +11,17 @@
     acl:mode        acl:Control;
     acl:accessTo    <./>;
     acl:default     <./>.
+
+<#minter-append-webids>
+    a acl:Authorization;
+    acl:agent <http://localhost:3002/dgt-id-proxy/card#me>;
+    acl:accessTo <./>;
+    acl:mode
+        acl:Read, acl:Append.
+
+<#minter-control-webids>
+    a acl:Authorization;
+    acl:agent <http://localhost:3002/dgt-id-proxy/card#me>;
+    acl:default <./>;
+    acl:mode
+        acl:Read, acl:Write, acl:Control.
```

Comment:
Since we'll be hosting WebIDs on a separate instance, this can probably be omitted in the pods package.

Reply:
To be re-added in a new package. (related: #534)

Reply:
I added this for local testing. Currently running start:watch will start the proxy as well, and it will be capable of creating profile documents so that functionality can be tested locally. It does so in the local pods package.

Do you want the proxy's local config to create profile documents on the separate instance? Because then we would be mixing test and production data. Unless we add a folder on to the pod where we will store the profile documents in which we can add the local test data.

Reply:
The (to be created) new package can also run locally like the current pods package, so no mixing of test/'prod' data. Just the same thing (locally) under a different name, might as well use it when it exists

Local proxy should point to local WebID server, live proxy to live WebID server at webid.netwerkdigitaalerfgoed.nl.

But probably something that should be done when the new package is created, to avoid having an invalid local proxy config in the meantime.